docs/RouteRequest.md

```diff
@@ -15,9 +15,9 @@ and in the [transferRequests in build-config.json](BuildConfiguration.md#transfe
 
 | Config Parameter                                                                                             |          Type          | Summary                                                                                                                                        |  Req./Opt. | Default Value    | Since |
 |--------------------------------------------------------------------------------------------------------------|:----------------------:|------------------------------------------------------------------------------------------------------------------------------------------------|:----------:|------------------|:-----:|
-| [alightSlack](#rd_alightSlack)                                                                               |       `duration`       | The minimum extra time after exiting a public transport vehicle.                                                                               | *Optional* | `"PT0S"`         |  2.0  |
+| [alightSlack](#rd_alightSlack)                                                                               |       `duration`       | The extra time after exiting a public transport vehicle.                                                                                       | *Optional* | `"PT0S"`         |  2.0  |
 | arriveBy                                                                                                     |        `boolean`       | Whether the trip should depart or arrive at the specified date and time.                                                                       | *Optional* | `false`          |  2.0  |
-| [boardSlack](#rd_boardSlack)                                                                                 |       `duration`       | The boardSlack is the minimum extra time to board a public transport vehicle.                                                                  | *Optional* | `"PT0S"`         |  2.0  |
+| [boardSlack](#rd_boardSlack)                                                                                 |       `duration`       | The extra time before boarding a public transport vehicle.                                                                                     | *Optional* | `"PT0S"`         |  2.0  |
```

Comment:
I guess this is a little tricky, because both the passenger and the vehicle can be the subject when we talk about boarding/alighting so we should be careful and make sure it is clear what we mean.

Reply:
https://github.com/opentripplanner/OpenTripPlanner/pull/5897/commits/657e62597e273bd2c0c181cc62f20c54675ef995